correlation_context/HTTP_HEADER_FORMAT.md

```diff
@@ -1,123 +1,98 @@
 
-When multiple headers are combined as described before, the resulting combined header will contain
-a version identifier for each individual header that was concatenated. This is intentional and parsers MUST NOT assume that any version field applies to the entire header value. After the first version identifier, another version identifier might occur at any time. Subsequent key/value pairs MUST be compliant with the denoted version until another version identifier is encountered or the end of the header value is reached.
 
+# Header Content
+The header contains a `list` of key-value pairs with optional properties attached.
 
 This section uses the Augmented Backus-Naur Form (ABNF) notation of [[!RFC5234]], including the DIGIT rule in <a data-cite='!RFC5234#appendix-B.1'>appendix B.1 for RFC5234</a>. It also includes the `OWS` rule from <a data-cite='!RFC7230#whitespace'>RFC7230 section 3.2.3</a>.
 
-The `DIGIT` rule defines numbers `0`-`9`.
-
-The `OWS` rule defines an optional whitespace character. To improve readability, it is used where zero or more whitespace characters might appear.
-
-The caller SHOULD generate the optional whitespace as a single space; otherwise, a caller SHOULD NOT generate optional whitespace. See details in the <a data-cite='!RFC7230#whitespace'>corresponding RFC</a>.
-
-The field value is a `list` of `list-members` separated by commas (`,`). A `list-member` is a key/value pair separated by an equals sign (`=`). Spaces and horizontal tabs surrounding `list-member`s are ignored. There can be a maximum of 180 `list-member`s in a `list`.
-One single key/value pair mut not be larger than `4096 Bytes`.
-The total size of the header MUST not exceed `8192 Bytes`.
+## Definition
 
-### list
-
-A simple example of the header value might look like:
-
-`v=0,key1=opaqueValue1[;properties1],key2=opaqueValue2[;properties2]`
-
-@TODO: Provide ABNF once agreed upon
+```
+list        = list-member 0*179( OWS "," OWS list-member )
+list-member = key OWS "=" OWS value *( OWS ";" OWS property )
+property    = key OWS "=" OWS value
+property    = key OWS
+key         = <token, defined in [[RFC2616], Section 2.2](https://tools.ietf.org/html/rfc2616#section-2.2)>
+value       = %x21 / %x23-2B / %x2D-3A / %x3C-5B / %x5D-7E
+              ; US-ASCII characters excluding CTLs,
+              ; whitespace DQUOTE, comma, semicolon,
+              ; and backslash
+OWS         = <Optional white space, as defined in [[RFC7230], Section 3.2.3.](https://tools.ietf.org/html/rfc7230#section-3.2.3)>
+```
 
-**Limits:**
+## Limits
 1. Maximum number of name-value pairs: `180`.
 2. Maximum number of bytes per a single name-value pair: `4096`.
 3. Maximum total length of all name-value pairs: `8192`.
 
-## Version
-A key value pair `v=<version>` MUST be the first element of the list.
-For this spec, the curent version is `0`.
+## Example
+`key1=value1[;property1;property2], key2 = value2[;property3]`
```

Comment:
Can we include a property with a value? like `property4=propertyvalue` or similar?

Reply:
fixed - the example also included [] which is wrong